.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,54 @@
+name: Bump Version and Start New Release
```

Comment:
a manually dispatched workflow to bump the minor version - triggers make-release.yml